lib/pmdtester/builders/pmd_report_builder.rb

```diff
@@ -36,13 +35,17 @@ def get_projects
       create_repositories_dir
 
       @projects.each do |project|
-        path = @repositories_dir + project.name
+        path = "#@repositories_dir/#{project.name}"
         clone_cmd = "#{project.type} clone #{project.connection} #{path}"
 
         Cmd.execute(clone_cmd) unless File::exist?(path)
```

Comment:
I would add some output (via `puts`) here, to know, what's going on, like "Skipping clone, project path #{path} already exists" or so. It helps to understand, what's going on.